NGS2_WITNESS_Cycle1_exploratory_exp1.R

```diff
@@ -0,0 +1,216 @@
+                             header = TRUE, sep = ',')
+empanelment <- read.csv('data/empanelment_cleaned.csv',
+                        header = TRUE, sep = ',')
+
+########################################################################################################################
+#Descriptive analyses - Cooperation (From Tian Zeng)
+########################################################################################################################
+
+### Number of unique players, sessions, conditions.
+
+length(unique(merged.exp1$empanel_id))
+length(unique(merged.exp1$sessionnum))
+unique(merged.exp1$condition)
+
+### Explore overall patterns of experimental sessions
+session_info=exp1_cooperation%>%
+  filter(round_num==1)%>%
+  group_by(sessionnum)%>%
+  summarise(
+    num_player=n(),
+    condition=unique(condition)[1]
+  )%>%
+  arrange(condition)
+
+## Display session_info
+datatable(session_info, 
+          caption = "Sessions: number of players and game conditions",
+          options = list(
+            scrollX = TRUE,
+            scrollCollapse = TRUE))
+
+#### The number of players vary within and between experimental conditions.
+ggplot(data = session_info, 
+       aes(x=num_player))+
+  geom_histogram(aes(weights=..count.., 
+                     fill=condition))+
+  scale_fill_brewer(palette="Set3")+
+  facet_wrap(~condition, ncol=1)
+
+#### Cooperation rate over the course of a game.
+
+session_round_rate=exp1_cooperation%>%
+  group_by(sessionnum, 
+           round_num)%>%
+  summarise(
+    rate_contr=mean(decision..0.D.1.C.)
+  )
+session_round_rate=left_join(session_round_rate, 
+                             session_info,
+                             by="sessionnum")
+
+ggplot(session_round_rate, 
+       aes(x=factor(round_num), 
+           y=rate_contr,
+           fill=condition))+
+  geom_boxplot()+
+  facet_grid(.~condition)+
+  labs(x="round")+
+  theme(axis.text.x=element_text(angle=0, 
+                                 vjust=0.4,
+                                 hjust=1))
+
+#### Association between behavior and social connections.
+
+coop_neighbor=
+  exp1_cooperation%>%
+  group_by(condition, 
+           num_neighbors, 
+           sessionnum)%>%
+  summarise(
+    rate_contr=mean(decision..0.D.1.C.)
+    )
+
+ggplot(coop_neighbor, 
+       aes(x=factor(num_neighbors), 
+           y=rate_contr,
+           fill=condition))+
+  geom_boxplot()+
+  facet_grid(.~condition)+
+  labs(x="number of neighbors")+
+  theme(axis.text.x=element_text(angle=0, 
+                                 vjust=0.4,
+                                 hjust=1))
+
+ggplot(exp1_cooperation, 
+       aes(x=factor(round_num), 
+           y=num_neighbors,
+           fill=condition))+
+  geom_boxplot()+
+  facet_grid(.~condition)+
+  labs(x="round")+
+  theme(axis.text.x=element_text(angle=0, 
+                                 vjust=0.4,
+                                 hjust=1))

```

Comment:
Looks like everything above is just printed to the console; none of this needs to be saved out to disk?